packages/components/psammead-headings/src/index.test.jsx

```diff
@@ -1,13 +1,18 @@
 import React from 'react';
 import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
-import { latin } from '@bbc/gel-foundations/scripts';
+import { latin, arabic } from '@bbc/gel-foundations/scripts';
 import { Headline, SubHeading } from './index';
 
 describe('Headline component', () => {
   shouldMatchSnapshot(
     'should render correctly',
     <Headline script={latin}>This is my headline.</Headline>,
   );
+
+  shouldMatchSnapshot(
+    'should render correctly with arabic script typography values',
+    <Headline script={arabic}>This is my headline.</Headline>,
```

Comment:
I think this should have arabic words

Reply:
I have just updated before your review 😄